src/config_cache.cpp

```diff
@@ -78,7 +78,6 @@ void config_cache::clear_defines()
 #ifdef __APPLE__
 	defines_map_["APPLE"] = preproc_define();
 
-#define MOUSE_TOUCH_EMULATION
```

Comment:
It was created for me to debug touch code on PC with a RMB - this way the development cycle is shorter.
I don't know yet, whether to remove it completely, or clean it up and add to PR. Maybe I'll leave it in a historical branch in my fork, as a reference.